client/my-sites/plugins/plugins-browser/style.scss

```diff
@@ -53,7 +53,7 @@
 .plugins-browser__main-container {
 	display: contents;
 
-	&::before {
+	.theme-default:not(.is-global) &::before {
```

Comment:
Unfortunately the `position: relative` to the parent container here doesn't fix things entirely, and we still get a shifted position of the content frame when going between the plugin and the plugin list. Removing this pseudo element from the global level seems ok.